packages/jest-fela-bindings/src/createSnapshotFactory.js

```diff
@@ -47,11 +46,8 @@ export default function createSnapshotFactory(
       div
     )
 
-    return (
-      formatCSS(renderToString(renderer)) +
-      os.EOL +
-      os.EOL +
-      formatHTML(div.innerHTML)
-    )
+    return `${formatCSS(renderToString(renderer))}\n\n${formatHTML(
```

Comment:
Platform-specific characters cannot be used in snapshots. The snapshot generated on Windows is not valid for Linux. I think most modern code editors understand a `\n` regardless of the operating system. I know what I'm talking about. I prefer to develop on Windows and this test constantly caused an error to me. It will be worse if the person generates snapshon on Windows and he has an error on the CI server.